diff --cc Assets/Scenes/moyu/level 1.unity
index 1207c7e,5cdeab5..0000000
--- a/Assets/Scenes/moyu/level 1.unity
+++ b/Assets/Scenes/moyu/level 1.unity
@@@ -333,6 -458,47 +333,50 @@@ CanvasRenderer
    m_PrefabAsset: {fileID: 0}
    m_GameObject: {fileID: 67154626}
    m_CullTransparentMesh: 1
++<<<<<<< HEAD
++=======
+ --- !u!4 &76000736 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+   m_PrefabInstance: {fileID: 3519430803959613576}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!4 &84040972 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+   m_PrefabInstance: {fileID: 577747593}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!1 &90164018
+ GameObject:
+   m_ObjectHideFlags: 0
+   m_CorrespondingSourceObject: {fileID: 0}
+   m_PrefabInstance: {fileID: 0}
+   m_PrefabAsset: {fileID: 0}
+   serializedVersion: 6
+   m_Component:
+   - component: {fileID: 90164019}
+   m_Layer: 0
+   m_Name: WarGround
+   m_TagString: Untagged
+   m_Icon: {fileID: 0}
+   m_NavMeshLayer: 0
+   m_StaticEditorFlags: 0
+   m_IsActive: 1
+ --- !u!4 &90164019
+ Transform:
+   m_ObjectHideFlags: 0
+   m_CorrespondingSourceObject: {fileID: 0}
+   m_PrefabInstance: {fileID: 0}
+   m_PrefabAsset: {fileID: 0}
+   m_GameObject: {fileID: 90164018}
+   serializedVersion: 2
+   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+   m_LocalPosition: {x: 32.25, y: 13.5, z: 0}
+   m_LocalScale: {x: 1, y: 1, z: 1}
+   m_ConstrainProportionsScale: 0
+   m_Children: []
+   m_Father: {fileID: 0}
+   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
  --- !u!1 &112132560
  GameObject:
    m_ObjectHideFlags: 0
@@@ -5939,37 -6304,63 +6080,97 @@@ Transform
    m_CorrespondingSourceObject: {fileID: 234218875943540166, guid: a2c9edb078fc23a41a8555fa1aec0dcc, type: 3}
    m_PrefabInstance: {fileID: 1357018253}
    m_PrefabAsset: {fileID: 0}
++<<<<<<< HEAD
 +--- !u!1 &579917529
 +GameObject:
 +  m_ObjectHideFlags: 0
 +  m_CorrespondingSourceObject: {fileID: 0}
 +  m_PrefabInstance: {fileID: 0}
 +  m_PrefabAsset: {fileID: 0}
 +  serializedVersion: 6
 +  m_Component:
 +  - component: {fileID: 579917530}
 +  m_Layer: 0
 +  m_Name: posB
 +  m_TagString: Untagged
 +  m_Icon: {fileID: 0}
 +  m_NavMeshLayer: 0
 +  m_StaticEditorFlags: 0
 +  m_IsActive: 1
 +--- !u!4 &579917530
 +Transform:
 +  m_ObjectHideFlags: 0
 +  m_CorrespondingSourceObject: {fileID: 0}
 +  m_PrefabInstance: {fileID: 0}
 +  m_PrefabAsset: {fileID: 0}
 +  m_GameObject: {fileID: 579917529}
 +  serializedVersion: 2
 +  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
 +  m_LocalPosition: {x: 5, y: 0, z: 0}
 +  m_LocalScale: {x: 1, y: 1, z: 1}
 +  m_ConstrainProportionsScale: 0
 +  m_Children: []
 +  m_Father: {fileID: 868145436}
 +  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
++=======
+ --- !u!1001 &577747593
+ PrefabInstance:
+   m_ObjectHideFlags: 0
+   serializedVersion: 2
+   m_Modification:
+     serializedVersion: 3
+     m_TransformParent: {fileID: 499075744}
+     m_Modifications:
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.x
+       value: 36
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.y
+       value: -10
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.z
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.w
+       value: 1
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.x
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.y
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.z
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.x
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.y
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.z
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7842041422370981350, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_Name
+       value: CinemachineCamera (3)
+       objectReference: {fileID: 0}
+     m_RemovedComponents: []
+     m_RemovedGameObjects: []
+     m_AddedGameObjects: []
+     m_AddedComponents: []
+   m_SourcePrefab: {fileID: 100100000, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
  --- !u!1 &583022692
  GameObject:
    m_ObjectHideFlags: 0
@@@ -6955,7 -7448,17 +7156,21 @@@ Transform
    m_CorrespondingSourceObject: {fileID: 234218875943540166, guid: a2c9edb078fc23a41a8555fa1aec0dcc, type: 3}
    m_PrefabInstance: {fileID: 2081448801}
    m_PrefabAsset: {fileID: 0}
++<<<<<<< HEAD
 +--- !u!1 &771920061
++=======
+ --- !u!4 &735833758 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+   m_PrefabInstance: {fileID: 1747031545}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!4 &751210061 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
+   m_PrefabInstance: {fileID: 794452970}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!1 &783030178
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
  GameObject:
    m_ObjectHideFlags: 0
    m_CorrespondingSourceObject: {fileID: 0}
@@@ -20955,6 -21495,16 +21170,19 @@@ Animator
    m_AllowConstantClipSamplingOptimization: 1
    m_KeepAnimatorStateOnDisable: 0
    m_WriteDefaultValuesOnDisable: 0
++<<<<<<< HEAD
++=======
+ --- !u!4 &996077062 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+   m_PrefabInstance: {fileID: 1732474536}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!4 &1009138906 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
+   m_PrefabInstance: {fileID: 842771876}
+   m_PrefabAsset: {fileID: 0}
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
  --- !u!1 &1019671378
  GameObject:
    m_ObjectHideFlags: 0
@@@ -21087,12 -21637,72 +21315,17 @@@ CanvasRenderer
    m_PrefabAsset: {fileID: 0}
    m_GameObject: {fileID: 1029904846}
    m_CullTransparentMesh: 1
 ---- !u!1001 &1039480034
 -PrefabInstance:
 -  m_ObjectHideFlags: 0
 -  serializedVersion: 2
 -  m_Modification:
 -    serializedVersion: 3
 -    m_TransformParent: {fileID: 139071121}
 -    m_Modifications:
 -    - target: {fileID: 4392751715906233192, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_Name
 -      value: PeaceBox
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalScale.x
 -      value: 0.5
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalScale.y
 -      value: 2
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalPosition.x
 -      value: 15
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalPosition.y
 -      value: 4.75
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalPosition.z
 -      value: 0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalRotation.w
 -      value: 1
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalRotation.x
 -      value: -0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalRotation.y
 -      value: -0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalRotation.z
 -      value: -0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalEulerAnglesHint.x
 -      value: 0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalEulerAnglesHint.y
 -      value: 0
 -      objectReference: {fileID: 0}
 -    - target: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 -      propertyPath: m_LocalEulerAnglesHint.z
 -      value: 0
 -      objectReference: {fileID: 0}
 -    m_RemovedComponents: []
 -    m_RemovedGameObjects: []
 -    m_AddedGameObjects: []
 -    m_AddedComponents: []
 -  m_SourcePrefab: {fileID: 100100000, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
 ---- !u!1 &1043960898
 +--- !u!4 &1053267491 stripped
 +Transform:
 +  m_CorrespondingSourceObject: {fileID: 234218875943540166, guid: a2c9edb078fc23a41a8555fa1aec0dcc, type: 3}
 +  m_PrefabInstance: {fileID: 938695662}
 +  m_PrefabAsset: {fileID: 0}
++--- !u!4 &1055031489 stripped
++Transform:
++  m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
++  m_PrefabInstance: {fileID: 369418010}
++  m_PrefabAsset: {fileID: 0}
 +--- !u!1 &1059102630
  GameObject:
    m_ObjectHideFlags: 0
    m_CorrespondingSourceObject: {fileID: 0}
@@@ -22256,6 -23089,68 +22546,71 @@@ SpriteRenderer
    m_SpriteTileMode: 0
    m_WasSpriteAssigned: 1
    m_SpriteSortPoint: 0
++<<<<<<< HEAD
++=======
+ --- !u!4 &1242082522 stripped
+ Transform:
+   m_CorrespondingSourceObject: {fileID: 5484313049908068699, guid: 30bb52d6b0682ad4fa6a62f144fb4309, type: 3}
+   m_PrefabInstance: {fileID: 1039480034}
+   m_PrefabAsset: {fileID: 0}
+ --- !u!1001 &1246890751
+ PrefabInstance:
+   m_ObjectHideFlags: 0
+   serializedVersion: 2
+   m_Modification:
+     serializedVersion: 3
+     m_TransformParent: {fileID: 499075744}
+     m_Modifications:
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.x
+       value: 36
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.y
+       value: -20
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalPosition.z
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.w
+       value: 1
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.x
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.y
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalRotation.z
+       value: -0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.x
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.y
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_LocalEulerAnglesHint.z
+       value: 0
+       objectReference: {fileID: 0}
+     - target: {fileID: 7842041422370981350, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+       propertyPath: m_Name
+       value: CinemachineCamera (2)
+       objectReference: {fileID: 0}
+     m_RemovedComponents: []
+     m_RemovedGameObjects: []
+     m_AddedGameObjects: []
+     m_AddedComponents: []
+   m_SourcePrefab: {fileID: 100100000, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
  --- !u!1001 &1252703516
  PrefabInstance:
    m_ObjectHideFlags: 0
@@@ -47340,143 -48009,16 +47695,33 @@@ CanvasRenderer
    m_PrefabAsset: {fileID: 0}
    m_GameObject: {fileID: 1423584782}
    m_CullTransparentMesh: 1
- --- !u!1 &1447662185
- GameObject:
-   m_ObjectHideFlags: 0
-   m_CorrespondingSourceObject: {fileID: 0}
-   m_PrefabInstance: {fileID: 0}
-   m_PrefabAsset: {fileID: 0}
-   serializedVersion: 6
-   m_Component:
-   - component: {fileID: 1447662188}
-   - component: {fileID: 1447662187}
-   - component: {fileID: 1447662186}
-   - component: {fileID: 1447662189}
-   m_Layer: 0
-   m_Name: Main Camera
-   m_TagString: MainCamera
-   m_Icon: {fileID: 0}
-   m_NavMeshLayer: 0
-   m_StaticEditorFlags: 0
-   m_IsActive: 1
- --- !u!81 &1447662186
- AudioListener:
-   m_ObjectHideFlags: 0
-   m_CorrespondingSourceObject: {fileID: 0}
-   m_PrefabInstance: {fileID: 0}
-   m_PrefabAsset: {fileID: 0}
-   m_GameObject: {fileID: 1447662185}
-   m_Enabled: 1
- --- !u!20 &1447662187
- Camera:
-   m_ObjectHideFlags: 0
-   m_CorrespondingSourceObject: {fileID: 0}
-   m_PrefabInstance: {fileID: 0}
-   m_PrefabAsset: {fileID: 0}
-   m_GameObject: {fileID: 1447662185}
-   m_Enabled: 1
-   serializedVersion: 2
-   m_ClearFlags: 1
-   m_BackGroundColor: {r: 0.19215687, g: 0.3019608, b: 0.4745098, a: 0}
-   m_projectionMatrixMode: 1
-   m_GateFitMode: 2
-   m_FOVAxisMode: 0
-   m_Iso: 200
-   m_ShutterSpeed: 0.005
-   m_Aperture: 16
-   m_FocusDistance: 10
-   m_FocalLength: 50
-   m_BladeCount: 5
-   m_Curvature: {x: 2, y: 11}
-   m_BarrelClipping: 0.25
-   m_Anamorphism: 0
-   m_SensorSize: {x: 36, y: 24}
-   m_LensShift: {x: 0, y: 0}
-   m_NormalizedViewPortRect:
-     serializedVersion: 2
-     x: 0
-     y: 0
-     width: 1
-     height: 1
-   near clip plane: 0.3
-   far clip plane: 1000
-   field of view: 60
-   orthographic: 1
-   orthographic size: 5
-   m_Depth: -1
-   m_CullingMask:
-     serializedVersion: 2
-     m_Bits: 4294967295
-   m_RenderingPath: -1
-   m_TargetTexture: {fileID: 0}
-   m_TargetDisplay: 0
-   m_TargetEye: 3
-   m_HDR: 1
-   m_AllowMSAA: 1
-   m_AllowDynamicResolution: 0
-   m_ForceIntoRT: 0
-   m_OcclusionCulling: 1
-   m_StereoConvergence: 10
-   m_StereoSeparation: 0.022
- --- !u!4 &1447662188
+ --- !u!4 &1439847807 stripped
  Transform:
-   m_ObjectHideFlags: 0
-   m_CorrespondingSourceObject: {fileID: 0}
-   m_PrefabInstance: {fileID: 0}
+   m_CorrespondingSourceObject: {fileID: 7076401142034430302, guid: 964b66ec36a5f6f43af83ddc2a56b6c3, type: 3}
+   m_PrefabInstance: {fileID: 1246890751}
    m_PrefabAsset: {fileID: 0}
++<<<<<<< HEAD
 +  m_GameObject: {fileID: 1447662185}
 +  serializedVersion: 2
 +  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
 +  m_LocalPosition: {x: 36, y: 0, z: -10}
 +  m_LocalScale: {x: 1, y: 1, z: 1}
 +  m_ConstrainProportionsScale: 0
 +  m_Children: []
 +  m_Father: {fileID: 0}
 +  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 +--- !u!114 &1447662189
 +MonoBehaviour:
 +  m_ObjectHideFlags: 0
 +  m_CorrespondingSourceObject: {fileID: 0}
 +  m_PrefabInstance: {fileID: 0}
++=======
+ --- !u!20 &1447662187 stripped
+ Camera:
+   m_CorrespondingSourceObject: {fileID: 6773061877011192318, guid: f8251ddcd86ec334ebc2ad3d81ad0887, type: 3}
+   m_PrefabInstance: {fileID: 8175315239978918205}
++>>>>>>> cd54208c26f949859464ee5ef0f25af171dec817
    m_PrefabAsset: {fileID: 0}
-   m_GameObject: {fileID: 1447662185}
-   m_Enabled: 1
-   m_EditorHideFlags: 0
-   m_Script: {fileID: 11500000, guid: a79441f348de89743a2939f4d699eac1, type: 3}
-   m_Name: 
-   m_EditorClassIdentifier: Unity.RenderPipelines.Universal.Runtime::UnityEngine.Rendering.Universal.UniversalAdditionalCameraData
-   m_RenderShadows: 1
-   m_RequiresDepthTextureOption: 2
-   m_RequiresOpaqueTextureOption: 2
-   m_CameraType: 0
-   m_Cameras: []
-   m_RendererIndex: -1
-   m_VolumeLayerMask:
-     serializedVersion: 2
-     m_Bits: 1
-   m_VolumeTrigger: {fileID: 0}
-   m_VolumeFrameworkUpdateModeOption: 2
-   m_RenderPostProcessing: 0
-   m_Antialiasing: 0
-   m_AntialiasingQuality: 2
-   m_StopNaN: 0
-   m_Dithering: 0
-   m_ClearDepth: 1
-   m_AllowXRRendering: 1
-   m_AllowHDROutput: 1
-   m_UseScreenCoordOverride: 0
-   m_ScreenSizeOverride: {x: 0, y: 0, z: 0, w: 0}
-   m_ScreenCoordScaleBias: {x: 0, y: 0, z: 0, w: 0}
-   m_RequiresDepthTexture: 0
-   m_RequiresColorTexture: 0
-   m_TaaSettings:
-     m_Quality: 3
-     m_FrameInfluence: 0.1
-     m_JitterScale: 1
-     m_MipBias: 0
-     m_VarianceClampScale: 0.9
-     m_ContrastAdaptiveSharpening: 0
-   m_Version: 2
  --- !u!1 &1454539309
  GameObject:
    m_ObjectHideFlags: 0
diff --git a/.vscode/settings.json b/.vscode/settings.json
index 63ba56c..e53ac25 100644
--- a/.vscode/settings.json
+++ b/.vscode/settings.json
@@ -42,7 +42,7 @@
         "**/*.mat": true,
         "**/*.meta": true,
         "**/*.prefab": true,
-        "**/*.unity": true,
+        "**/*.unity": false,
         "build/": true,
         "Build/": true,
         "Library/": true,
